src/ngNotificationsBar.js

```diff
@@ -153,6 +158,10 @@
 					notificationHandler(event, data, 'success');
 				});
 
+				scope.$on('notifications:closeAll', function () {
+					notifications.length = 0;
```

Comment:
I assumed this works otherwise you wouldn't have made a pull request, but would it just be better to set notifications to an empty array?

`notifications = [];`